pypuf/experiments/experimenter.py

```diff
@@ -41,7 +41,10 @@ def run(self):
 
             # define experiment process
             def run_experiment(queue, semaphore, logger_name):
-                exp.execute(queue, logger_name)  # run the actual experiment
+                try:
+                    exp.execute(queue, logger_name)  # run the actual experiment
+                except:
+                    print('ERROR Experiment failed')
```

Comment:
Do not 'swallow' exception information, but pass it on to the user. Otherwise, debugging is gonna be very hard. How about

    except e:
        semaphore.release()
        raise e